src/account/accountSelectors.js

```diff
@@ -1,5 +1,5 @@
 export const getActiveAccount = (state) =>
-  state.account[0];
```

Comment:
No they're actually slightly different. If `state.account` is the empty list (`[]`), then trying to access element 0 will throw an exception